types/src/access_path.rs

```diff
@@ -161,6 +161,14 @@ impl AccessPath {
         }
     }
 
+    /// Extracts a [ModuleId]. Returns [None] if this is not a module access.
+    pub fn get_module_id(&self) -> Option<ModuleId> {
```

Comment:
nit: `try_get_module_id`?

Reply:
Called that to be consistent with struct tag getter, but I guess we should not be consistent with bad naming😄 done